### DISCUSSION TITLE
Can This Be Used Without A Domain?

### DISCUSSION THREAD

**Discussion by kn4thx** [hwgilbert16/scholarsome#163] [Q&A]
I have a few old slim clients I run some things on locally and was wondering if I can omit a host name. I also ran into an issue upon deployment where I am getting an error 

"Datasource "db": MySQL database at "scholarsome"
Error: P1001: Can't reach database server at `scholarsome`:`3306`"

and was curious if this is because the domain I put in the .env variable isn't set up yet. I have verified that the DB is running on port 3306 in the logs so was troubleshooting why it cannot connect.

**Comment by kvaml** [comment]
Would localhost work where you have scholarsome in your config?  Or add scholarsome with its IP address to your hosts file?
